grafana-plugin/src/pages/routes.tsx

```diff
@@ -17,21 +14,6 @@ import LiveSettingsPage from 'pages/settings/tabs/LiveSettings/LiveSettingsPage'
 import Test from 'pages/test/Test';
 import UsersPage from 'pages/users/Users';
 
-export interface NavMenuItem {
-  meta: AppRootProps['meta'];
-  pages: { [id: string]: PageDefinition };
-  path: string;
-  page: string;
-  grafanaUser: {
-    orgRole: 'Viewer' | 'Editor' | 'Admin';
-  };
-  enableLiveSettings: boolean;
-  enableCloudPage: boolean;
-  enableNewSchedulesPage: boolean;
-  backendLicense: string;
-  onNavChanged: any;
-}
```

Comment:
I removed this because I saw a reference to `grafanaUser.orgRole`. I then searched in both `oncall` and `oncall-private` repos and saw no reference to `NavMenuItem`, therefore I think it is safe/makes sense to remove it.